### PR TITLE
[EDU-3142] feat: ad tune waf guide

### PR DESCRIPTION
**Pull request by hannah-sk**
### Changes

* Added new guide on How to tune WAF.

[EDU-3142]: https://aziontech.atlassian.net/browse/EDU-3142?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/en/pages/secure-journey/advanced-configurations/tune-waf.mdx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+1. On **Domain(s)**, select `www.mydomain.com` and `www.anotherdomain.com`.
+2. On **Time Range**, select **Last 12 hours**.
+3. On **Network List**, select **Blocklist IPs**.
+4. Click **Apply filter**.
+
+You'll receive a list of records regarding the filters you've applied. In this case, records that match the `Blocklist IPs` network list.
+
+You can do a drilldown of the records by clicking on each of them with information on **Hits**, **IPs**, **Countries**, **Top 10 IPs Address**, **Top 10 Countries**, and **Top 10 Paths**.
+
+## Creating an Allowed Rules
```
Comment (bru-andrade) [inline src/content/docs/en/pages/secure-journey/advanced-configurations/tune-waf.mdx:46]:
```suggestion
## Creating an Allowed Rule
```

File: src/content/docs/en/pages/secure-journey/advanced-configurations/tune-waf.mdx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    - Under the **Filter Possible Attacks** section, you'll find the available variables to filter.
+3. In the **Domain(s)** dropdown, select the domains you want to analyze. Mandatory field.
+4. Optionally, you can choose to filter by:
+    - Time range
+    - Network List
+    - IP Address
+    - Country
+5. Click the **Apply filter** button.
+
+To exemplify a real-life situation, let's assume the existence and analysis of few variables:
```
Comment (bru-andrade) [inline src/content/docs/en/pages/secure-journey/advanced-configurations/tune-waf.mdx:35]:
```suggestion
To exemplify a real-life situation, let's assume the existence of a few variables:
```

**Review by bru-andrade — APPROVED** [review]
Lindão ❤️

File: src/content/docs/en/pages/secure-journey/advanced-configurations/tune-waf.mdx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+- <Badge variant="accent">[Configured edge firewall]()</Badge><br></br>
+
+- <Badge variant="accent">[Activated WAF in the edge firewall](/en/documentation/products/guides/secure/configure-main-settings/#modules)</Badge>
+
+After setting up the initial configurations, proceed as follows:
+
+1. [Access RTM](/en/documentation/products/guides/how-to-access-rtm/) > select **WAF Rules**.
+2. Select the **Tuning** tab.
+    - Under the **Filter Possible Attacks** section, you'll find the available variables to filter.
+3. In the **Domain(s)** dropdown, select the domains you want to analyze. Mandatory field.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/secure-journey/advanced-configurations/tune-waf.mdx:27]:
```suggestion
3. In the **Domain(s)** dropdown, select the domains you want to analyze. It's a **mandatory field**.
```

File: src/content/docs/en/pages/secure-journey/advanced-configurations/tune-waf.mdx [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+menu_namespace: secureMenu
+---
+
+import Badge from '~/components/Badge.astro'
+
+You can use the [WAF Tuning](/en/documentation/products/edge-firewall/web-application-firewall/#waf-tuning) feature to analyze network lists, IPs, and countries that have matched configured WAF rule sets.
+
+In this guide, you'll filter possible attacks in your WAF and create an allowed rule from the results you receive.
+
+Requires:
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/secure-journey/advanced-configurations/tune-waf.mdx:16]:
```suggestion

---

## Filtering data

Requires:
```

Comment (MarianaAguilera) [reply src/content/docs/en/pages/secure-journey/advanced-configurations/tune-waf.mdx:16]: idk if it's the best title, but suggestion de separate better the topics.

**Review by MarianaAguilera — APPROVED** [review]
Left a few suggestions

**Pull request merged**